core/jazz_ui/src/app/core/constants/field-name-mapping.ts

```diff
@@ -0,0 +1,10 @@
+// Note: LHS should be in lowercase. 
+
+export const MAPPING = {
+    'apigateway': 'API Gateway',
```

Comment:
Update to complete list.

Reply:
Complete list: https://github.com/tmobile/jazz/blob/sls_app/core/jazz_assets/swagger/swagger.json#L131

Reply:
From the list, following assets need mapping: 
cloudwatch_event => `CloudWatch Event`
apigee_proxy => `Apigee Proxy`
kinesis_stream => `Kinesis Stream`